### DISCUSSION TITLE
Website: Any way to contribute to updating the Comparison page?

### DISCUSSION THREAD

**Discussion by redbar0n** [urql-graphql/urql#1278] [Q&A] [answered]
Q1: I was wondering if there was a way to submit a PR to update the comparison page on the website?
https://formidable.com/open-source/urql/docs/comparison/

Without community contributions it would probably be hard to keep track of the other projects as they evolve.
See example below.

React Query has a very similar Comparison page (don't know who came first), but they allow PR's:
https://react-query.tanstack.com/comparison

Q2: Might also be an idea to submit a PR to them so URQL is included in the comparison?
Q3: And include React Query on the URQL comparison page too?

Example:
https://formidable.com/open-source/urql/docs/comparison/ says that Relay does NOT have "stale time configuration".
https://react-query-git-ce4fe79195bc3c2d81a564e80473188eb00f62e8.tannerlinsley.vercel.app/docs/comparison (a pending PR) says that Relay HAS "stale time configuration".

Q4: I don't know which is true. Do you?
Q5: Since neither page has a date stamp, neither do I know which is the most up to date page. Do you?

**Comment by JoviDeCroock** [comment, marked as answer]
Our documentation page lives here https://github.com/FormidableLabs/urql/blob/main/docs/comparison.md.

I guess a big difference between `react-query` and our page is that we are comparing in-depth on GraphQL feature as compared to general data-fetching. For stuff like caching and ssr it might be worth but generally I feel like both feed other purposes.

Afaik there's no way to configure that in Relay (happy to be proven differently), I do know there's something like a maximum amount of operations to be kept in-memory but that's about it.

EDIT: https://github.com/facebook/relay/blob/master/packages/relay-runtime/store/RelayModernStore.js#L140 this might be an indication of it existing. When composing this page we mainly went off documentation, this feature seems undocumented but exists in the codebase so I guess it _is_ supported.

In terms of the staleness, I guess it would be a good thing to include a `last Update at: X` which can be done manually or retrieved from Git.

**Comment by kitten** [comment]
All the docs are in the repo and PRs are welcome: https://github.com/FormidableLabs/urql/blob/main/docs/comparison.md ✌️

We also sure could include it but someone was volunteering to add it; it's not really high on our priority list though 😅

We have explicitly not included React Query in our Comparison list as it's not a fully fledged GraphQL client. It works in the same way as any other REST-based state/fetch manager does and has features that go well with GraphQL, but we drew the line at the most commonly known and chosen GraphQL clients.

In terms of the Relay information; we go by what's on their website, so discrepancies exist. We've talked to people that use Relay frequently and ones that contribute to its ecosystem and the page is what we then came up with, including a lot of changes. I can't vouch for its accuracy 100% at all times, but I haven't seen any reference to stale-time configuration, i.e. TTL-based (or otherwise) refetches on their docs. Generally, information on Relay is hard to gather because of their docs state, so arguably if a user can't find information on a feature even if it exists, is it usable and accessible? 😅

Hope that makes sense! 💖
